src/ecs/resources/input.rs

```diff
@@ -333,9 +333,9 @@ impl InputHandler {
     }
 
     /// Returns the value of an axis by the i32 id, if the id doesn't exist this returns None.
-    pub fn axis_value(&self, id: i32) -> Option<f32> {
+    pub fn axis_value(&self, id: &str) -> Option<f32> {
```

Comment:
Good point, I looked into it and I must have indeed been thinking about `Into<String>`. But my first point still stands. Even the Rust standard library makes frequent use of `AsRef` in exactly this way to avoid such boilerplate, and I think that users might appreciate an API that assists with zero runtime cost.